src/types/profile.ts

```diff
@@ -90,7 +90,7 @@ export function toAppProfileWithEmail(
     name: user.name,
     preferred_email: user.preferred_email,
     preferred_languages: from.preferredLanguages,
-    version: from.version
+    version: from.version as NonNegativeNumber
```

Comment:
for `0` you are allowed to cast to `NonNegativeNumber` since you're sure that `0` is a `NonNegativeNumber`

in this case though, you can't cast, since `from` comes from an eternal source, you have to validate it first